src/backendMiddleware.ts

```diff
@@ -38,6 +41,16 @@ export class BackendMiddleware {
         gateway: jolocomContractsGateway,
       },
     })
+    if (SKIP_IDENTITY_REGISTRATION) {
+      this.registry.commit = this.fuelKeyWithEther = async (arg: any) =>
+        undefined
+      this.setIdentityWallet = async function(
```

Comment:
I'm not sure if I'm getting the implications of this change: This creates always a new identity when the app starts without initializing the app first, isn't it?

Reply:
No it doesn't create a new identity every time. If there is encrypted entropy already stored in the DB it is reused (through `checkIdentityExists`). The difference is that the `identityWallet` is created through `registry.create` instead of `registry.authenticate` to avoid anchoring the identity.

Maybe I need to add some comments to the config file!

Reply:
Ah ok, and `create` is not anchoring the identity because `registry.commit` it overwritten, but still returning the identityWallet.
And `checkIdentityExists` is also going this way with the data from the DB.